packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -18,7 +18,7 @@ Terra.describeViewports('Date Picker', ['medium'], () => {
       browser.click('[class*="button"]');
     });
 
-    Terra.it.matchesScreenshot({ selector: '[class="react-datepicker"]' });
+    Terra.it.matchesScreenshot({ selector: '#calendar' });
```

Comment:
Left some [comments here](https://github.com/cerner/terra-framework/pull/794#discussion_r312166013) to switch over to a data attribute so we don't run into duplicate ID issue if people render multiple date pickers on the same page. We'd want to update this as well.